bot/resources/tags/dotenv.md

```diff
@@ -14,10 +16,3 @@ from dotenv import load_dotenv()
 
 load_dotenv(".env")
 ```
-The variables from the file have now been loaded into your programs environment, and you can access them using `os.getenv()` anywhere in your program, like this:
-```py
-from os import getenv
-
-my_token = getenv("TOKEN")
-```
-For further reading about tokens and secrets, please read [this explanation](https://vcokltfre.dev/tips/tokens).
```

Comment:
Why was this section removed?

Reply:
I am really sorry it might have been accidentally removed by the script.